web/backend_api_builder.py

```diff
@@ -70,3 +70,11 @@ def case_id(self, case_id):
         :return: case endpoint
         """
         return self.cases + '/{}'.format(case_id)
+
+    def case_list(self,beneficiary_id):
+        """
+        Generates endpoint for retrieving/updating task with given cases_id
+        :param cases_id: unique cases id
+        :return: case endpoint
+        """
+        return self.case_id(beneficiary_id) + '/beneficiary_id?beneficiary_id={}'.format(beneficiary_id)
```

Comment:
this is also wrong .. the final api should be 
cases?beneficiary_id?beneficiary_id={}